src/Build/Resources/Strings.resx

```diff
@@ -332,6 +332,13 @@
   <data name="TryingExtensionsPath" xml:space="preserve">
     <value>Trying to import {0} using extensions path {1}</value>
   </data>
+  <data name="DuplicateOverrideUsingTaskElement" xml:space="preserve">
+    <value>MSB4275: Attempted to create multiple overrides of the same task: {0}</value>
```

Comment:
Can you show an example of this message? Is it possible to point to both locations?

Reply:
There doesn't seem to be a convenient way to get location from a `RegisteredTaskRecord`, which is the only reference we have to the other usingtask at the point that this is logged.

Reply:
Here's the log after a unit test:

```
Build started.
Evaluation started ("C:\Users\bevillal\AppData\Local\Temp\pm3tsjgb.5xz\Temporary458011a3c99d422e8e03cf88a5710645\build.proj")
Created an override using task: Foo at C:\Users\bevillal\AppData\Local\Temp\pm3tsjgb.5xz\Temporary458011a3c99d422e8e03cf88a5710645\build.proj (1,89)
C:\Users\bevillal\AppData\Local\Temp\pm3tsjgb.5xz\Temporary458011a3c99d422e8e03cf88a5710645\build.proj(1,186):  error MSB4275: Attempted to create multiple overrides of the same task: Foo
Evaluation finished ("C:\Users\bevillal\AppData\Local\Temp\pm3tsjgb.5xz\Temporary458011a3c99d422e8e03cf88a5710645\build.proj")
Project "build.proj" (default targets):
Target "Bar" in project "C:\Users\bevillal\AppData\Local\Temp\pm3tsjgb.5xz\Temporary458011a3c99d422e8e03cf88a5710645\build.proj" (entry point):
Done building target "Bar" in project "build.proj".
Done building project "build.proj".
Build succeeded.
```

Reply:
This is confusing since they're on the same line but I think that's just an artifact of the test construction?

In any case there are messages for each that have a location so I'm pretty good with that.